network-protection/network-protection-impl/src/main/java/com/duckduckgo/networkprotection/impl/subscription/NetpSubscriptionManager.kt

```diff
@@ -70,9 +74,21 @@ class RealNetpSubscriptionManager @Inject constructor(
 
     private fun hasValidEntitlementFlow(): Flow<Boolean> = subscriptions.getEntitlementStatus().map { it.contains(NetP) }
 
-    private suspend fun getVpnStatusInternal(hasValidEntitlement: Boolean): VpnStatus {
+    private suspend fun getVpnStatusInternal(hasValidEntitlement: Boolean): VpnStatus = if (newSettingsFeature.self().isEnabled()) {
```

Comment:
Please add brackets to the method. I know kotlin allows for this but it harms readability.